docker/gpu/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Prepare the image with:
+#   docker build -t tensorflow/haskell:1.0.1-gpu docker/gpu
+FROM gcr.io/tensorflow/tensorflow:1.0.1-gpu
+MAINTAINER TensorFlow authors <tensorflow-haskell@googlegroups.com>
+
+RUN apt-get update
+
+RUN apt-get install -y \
+        # stack needs git
+		git \
+        # Required by snappy-frames dependency.
+        libsnappy-dev \
+		# Avoids /usr/bin/ld: cannot find -ltinfo
+	    libncurses5-dev \
+	    # Makes stack viable in the container
+	    libgmp-dev \
+	    # Required for locales configuration.
+	    locales
+
+# Support output of Unicode characters.
+RUN dpkg-reconfigure locales && \
+    locale-gen en_US.UTF-8 && \
+	    update-locale LANG=en_US.UTF-8
```

Comment:
Can you fix the indentation here and above?

Reply:
Btw, I'not too happy about the duplication in the Dockerfiles for this. Is there a reason the protoc and libraries are installed with curl, rather than using an upstream devel tensorflow (like 1.3.0-devel-gpu)?

I am in the process of testing your upgrade (#161) to 1.3 now, and will update this PR as necessary.